packages/components/Tabs/src/Tabs.types.ts

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import { ITextProps } from '@fluentui-react-native/text';
+import type { IViewWin32Props } from '@office-iss/react-native-win32';
+import { IRenderData } from '@uifabricshared/foundation-composable';
+import { IForegroundColorTokens, FontTokens, IBackgroundColorTokens } from '@fluentui-react-native/tokens';
+import { FocusZoneProps } from '@fluentui-react-native/focus-zone';
+import { View, ViewProps } from 'react-native';
+
+export const tabsName = 'Tabs';
+
+export interface ITabsContext {
```

Comment:
Is it possible to name this `TabsContext` instead of `ITabsContext`?

Reply:
On Tabs.tsx, line 15, we bumped into the need of having two variables with the same name, se we kept the "I" for that one:
`export const TabsContext = React.createContext<ITabsContext>({...});`

We are open to any suggestions for renaming them.

Reply:
Maybe something like `TabsContextData`?